charts/nutanix-csi-storage/README.md

```diff
@@ -161,12 +161,15 @@ Install the driver in the `ntnx-system` namespace and create a volume storagecla
 ```console
 helm install nutanix-storage nutanix/nutanix-csi-storage -n ntnx-system --create-namespace --set volumeClass=true --set prismEndPoint=X.X.X.X --set username=admin --set password=xxxxxxxxx --set storageContainer=container_name --set fsType=xfs
 ```
+In the above command  `prismEndpoint` refers to the Prism Element cluster virtual ip address where k8s vms are running. 
```

Comment:
agree for the first part to clarify we are talking about Prism Element cluster virtual ip, but k8s vm can run anywhere not only on this cluster so let's replace by something like

`In the above command prismEndpoint refers to the Prism Element cluster virtual ip address where storage will be consumed.`